crates/nu-command/src/system/run_external.rs

```diff
@@ -85,9 +85,34 @@ impl Command for External {
             return Ok(PipelineData::new(call.head));
         }
 
+        let mut args_strs = vec![];
+
+        for arg in args {
+            if let Ok(s) = arg.as_string() {
+                args_strs.push(s);
+            } else if let Value::List { vals, .. } = arg {
+                // Interpret a list as a series of arguments
+                for val in vals {
+                    if let Ok(s) = val.as_string() {
+                        args_strs.push(s);
+                    } else {
```

Comment:
Im wondering if this should be a recursive function

Reply:
Yeah, could be. It would have a side effect though that it would work for nested lists as well. It might be more desirable to support only flat lists? Since in case of a nested list the chance is you have an error in your arguments.